npg_qc_viewer/root/src/about_qc_checks.tt2

```diff
@@ -10,12 +10,17 @@
 
 <div id="content">
 
- <a name="tag_metrics"></a>
+ <a name="tag metrics_check"></a>
  <h3>Tag metrics</h3>
```

Comment:
Please consider the following text:
Please consider the following text:

This check displays statistics about decoding of the index read. The semantics of the summary table data differs for pools and individual libraries.

For a pool the overall decoding percent is displayed followed by the coefficient of variance that characterises the uniformity of tag distribution in a pool. The background of the table cell is blue if the overall decoding rate is over 80%, for lesser values teh background is red.

For an individual library the decoding percent for this library withing a pool is displayed. If the value is below XXX the background is red, otherwise it is grey.

Then the previously available CV calculation.